### PR TITLE
chore: Change Production Release action item sequence for cht-android

### PR DESCRIPTION
**Pull request by sugat009**
# Description

1. Change the Production Release action item sequence i.e. put the create CHANGELOG action item before creating the tag.
2. A few grammatical fixes

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

File: content/en/contribute/code/android/releasing.md [hunk 1]
```diff
@@ -21,17 +21,17 @@ All Medic's android projects automatically build, sign, and release builds via G
...
+3. Creating the tag will trigger the building and signing of the app in CI. The release-ready APKs are available for side-loading from the Releases section in the project (e.g. [CHT-Android Releases](https://github.com/medic/cht-android/releases)), along with the AABs that the Google Play Store may require. Note that the created release on GitHub with the generated artifacts will not be explicitly linked to the tag because the release is in a draft state.
 4. **Side-Load**: for testing internally, or apps that are not published in the Play Store like Collect and Gateway: Navigate to the GitHub Releases page (linked above) and download the relevant APKs for distribution.
 5. Announce the release in _#quality-assurance_.
 
 
 ## Production release
 
-1. Repeat step 2-3 from the above [alpha release section](#alpha-for-release-testing) with the naming convention `v<major>.<minor>.<patch>`.
 1. Update the CHANGELOG with the details of what's in this release.
```
Comment (latin-panda) [inline content/en/contribute/code/android/releasing.md:33]: Can you please fix the numeration, all these are `1.`?

Comment (sugat009) [reply content/en/contribute/code/android/releasing.md:33]: When rendering, markdown changes them to appropriate numbers according to the sequence.

Comment (latin-panda) [reply content/en/contribute/code/android/releasing.md:33]: Oh interesting. I guess it's just OCD haha

**Review by latin-panda — APPROVED** [review]
Thanks!

**Pull request merged**